auth/api/iam/api.go

```diff
@@ -71,6 +75,15 @@ type Wrapper struct {
 	policyBackend policy.PDPBackend
 	templates     *template.Template
 	storageEngine storage.Engine
+	keyStore      crypto.KeyStore
+}
+
+type Oid4vciSession struct {
+	HolderDid   string
+	IssuerDid   string
+	RedirectUrl string
+	RedirectUri string
```

Comment:
Describe the difference

Reply:
Done, renamed fields with more descriptive names and added documentation.